tests/page-objects/forms/pregnancy-visit-form.wdio.page.js

```diff
@@ -20,8 +20,8 @@ const dangerSignSummary = () =>
   $$('label.question.readonly.or-branch.non-select.or-appearance-li');
 const followUpMessage = () => $('[data-value=" /pregnancy_visit/chw_sms "]');
 
-const selectPatient = (patientName) => {
-  return genericForm.selectContact('/pregnancy_visit/inputs/contact', patientName);
+const selectPatient = async (patientName) => {
+  await genericForm.selectContact('/pregnancy_visit/inputs/contact', patientName);
```

Comment:
I think I added something following that and changed it, but then moved the code to somewhere else. 
In async/await I prefer not to return a pending promise, because if the promise fails, it ends up being reported in the caller instead of this scope. 
Are you very much against this change ? This doesn't actually fix anything, it's just syntax and better error reporting.

Reply:
> Are you very much against this change ? 

No, it's very minor. I just imagine someone changing it back in the future just to make it compact, both work and either way the stack trace shows where the error happened